### PR TITLE
Proper natspec, comments and sequence

### PR DESCRIPTION
**Pull request by parv3213**

### CODE REVIEW

File: contracts/HomeFi.sol [hunk 1]
```diff
@@ -9,25 +9,37 @@ import {ERC721URIStorageUpgradeable} from "@openzeppelin/contracts-upgradeable/t
 import {ContextUpgradeable, ERC2771ContextUpgradeable} from "@openzeppelin/contracts-upgradeable/metatx/ERC2771ContextUpgradeable.sol";
 
 /**
```
[the remark this answers is not in the document]
Comment (parv3213) [reply contracts/HomeFi.sol:11]: For example, external and public functions of Project contract are defined in IProject. Now, should we all `@ inheritdoc` to all Project functions? I am pretty sure `inheritdoc ` is already implied when we use the `override` keyword.

File: contracts/HomeFi.sol [hunk 2]
```diff
@@ -134,37 +169,50 @@ contract HomeFi is
         override
         onlyAdmin
     {
+        // TODO add HomeFi::!Change?
```
[the remark this answers is not in the document]
Comment (parv3213) [reply contracts/HomeFi.sol]: Added here: https://github.com/RigorHQ/Rigor-ProtocolV2/pull/203/commits/916ab3f0ec8e202756094248de65c887e54b2df1

File: contracts/HomeFi.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (zgorizzo69) [reply contracts/HomeFi.sol:11]: Yes we would need @inheritdoc for all project functions to copy the doc from the base class
ex : 
in interface
```
  /// @notice Burn a meToken to receive the underlying asset
    /// @param meToken          Address of meToken to burn
    /// @param meTokensBurned   Amount of meTokens to burn
    /// @param recipient        Address to receive the underlying assets
    function burn(
        address meToken,
        uint256 meTokensBurned,
        address recipient
    ) external;
    ```
    in implementation 
    ```
   /// @inheritdoc IFoundryFacet
    function burn(
        address meToken,
        uint256 meTokensBurned,
        address recipient
    ) external override {
      ....
    }
     
    ```

[the remark this answers is not in the document]
Comment (parv3213) [reply contracts/HomeFi.sol:11]: Gotcha

[the remark this answers is not in the document]
Comment (parv3213) [reply contracts/HomeFi.sol:11]: Added here : https://github.com/RigorHQ/Rigor-ProtocolV2/pull/203/commits/a0885b44c81d43033c4c2426197729aaa7cea292

**Comment by parv3213** [comment]
> version

Yeah, I think so too. As the project contract isn't upgradable. So we never need to change its version without a new deployment.

**Comment by zgorizzo69** [comment]
> @zgorizzo69 Additional changes required in subgraph, UI or bot:
> 
>     1. Rename Project's contract `version` to `VERSION`.
> 
>     2. Rename Project's contract `fundProject` to `allocateFunds` in incomplete allocation bot.

I will create a PR right away

**Pull request merged**